### PR TITLE
Add function to get directory of a materialized layer within a window

### PR DESCRIPTION
**Pull request by favyen2**
Addresses the most obvious TODO in #103 

Note this PR depends on #104.

### CODE REVIEW

File: rslearn/dataset/manage.py [hunk 1]
```diff
@@ -198,8 +198,8 @@ def materialize_window(
         layer_cfg: the layer config
     """
     # Check if layer is materialized already.
-    completed_fname = window.path / "layers" / layer_name / "completed"
-    if completed_fname.exists():
+    layer_dir = window.get_layer_dir(layer_name)
+    if (layer_dir / "completed").exists():
```
Comment (Hgherzog) [inline rslearn/dataset/manage.py:202]: nit: perhaps have a is_complet utility function that just checks for a completed file

File: rslearn/dataset/window.py [hunk 2]
```diff
@@ -13,6 +13,47 @@
 
 logger = get_logger(__name__)
 
+LAYERS_DIRECTORY_NAME = "layers"
```
Comment (Hgherzog) [inline rslearn/dataset/window.py:16]: I would recommend adding some tests for these directory structure functions

**Review by Hgherzog — APPROVED** [review]
LGTM, a couple minor suggestions. Thanks for doing this it is really helpful!

File: rslearn/dataset/manage.py (hunk 1 above)
Comment (favyen2) [reply rslearn/dataset/manage.py]: added

File: rslearn/dataset/window.py (hunk 2 above)
Comment (favyen2) [reply rslearn/dataset/window.py:16]: added a few tests. some of them will just throw error if we change the directory structure, while the one for mark_layer_completed/is_layer_completed will actually test logic.

**Pull request merged**